`GenericMap.draw_contour()` makes use of the `plot_settings` dictionary (#7844)
sunpy/sunpy: changelog/7844.bugfix.rst

> Fixed a bug where custom values in the ``plot_settings`` dictionary were not being correctly applied in the :meth:`sunpy.map.GenericMap.draw_contours` method, ensuring proper propagation of settings to contour plots.

Reviewer: ```suggestion
Fixed custom values in the ``plot_settings`` dictionary not being correctly applied by :meth:`sunpy.map.GenericMap.draw_contours`.
```

> Fixed a bug where custom values in the ``plot_settings`` dictionary were not being correctly applied in the :meth:`sunpy.map.GenericMap.draw_contours` method, ensuring proper propagation of settings to contour plots.

Reviewer: ```suggestion
Fixed a bug where custom values in the ``plot_settings`` dictionary were not being correctly applied in the :meth:`sunpy.map.GenericMap.draw_contours` method.
```
